swarm/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.45.0 -- unreleased
+
+- Implement refactored `Transport`.
+  See [PR 4568]
+- Move `address_translation` into swarm.
+  See [PR 4568]
```

Comment:
Not just that, we also move the functionality to `libp2p-identify`. This should be called out in the changelog. Not everyone might be using `libp2p-identify` so they might lose the ability of address translation without it.

Probably also worth calling out in the main changelog.